hello-greenfox/src/Display.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Display = ({ counter }) => (<div> {counter} </div>);
```

Comment:
The project discription says:
Display should get a number via the special children property — containment

I don't see it :/